exporting/prometheus/multi_metadata.yaml

```diff
@@ -0,0 +1,436 @@
+          - name: 'send charts matching'
+            default_value: '*'
+            description: |
+              One or more space separated patterns (use * as wildcard) checked against both chart id and chart name.
+            required: false
+            detailed_description: |
+              A pattern starting with ! gives a negative match. So to match all charts named apps.* except charts ending in *reads,
+              use !*reads apps.* (so, the order is important: the first pattern matching the chart id or the chart name will be used,
+              positive or negative). There is also a URL parameter filter that can be used while querying allmetrics. The URL parameter
+              has a higher priority than the configuration option.
+          - name: 'send names instead of ids'
+            default_value: ''
+            description: 'Controls the metric names Netdata should send to the external database (yes|no).'
+            required: false
+            detailed_description: |
+              Netdata supports names and IDs for charts and dimensions. Usually IDs are unique identifiers as read by the system and names
+              are human friendly labels (also unique). Most charts and metrics have the same ID and name, but in several cases they are
+              different : disks with device-mapper, interrupts, QoS classes, statsd synthetic charts, etc.
+          - name: 'send configured labels'
+            default_value: ''
+            description: 'Controls if host labels defined in the `[host labels]` section in `netdata.conf` should be sent to the external database (yes|no).'
+            required: false
+          - name: 'send automatic labels'
+            default_value: ''
+            description: 'Controls if automatically created labels, like `_os_name` or `_architecture` should be sent to the external database (yes|no).'
+            required: false
+      examples:
+        folding:
+          enabled: true
+          title: ''
+        list:
+          - name: 'Example configuration'
+            folding:
+              enabled: false
+            description: 'Basic example configuration for Prometheus remote write.'
+            config: |
+             [prometheus_remote_write:my_instance]
+                 enabled = yes
+                 destination = example.domain:example_port
+                 remote write URL path = /receive
+          - name: 'Example configuration with HTTPS and HTTP authentication'
+            folding:
+              enabled: false
+            description: 'Add `:https` modifier to the connector type if you need to use the TLS/SSL protocol. For example: `remote_write:https:my_instance`.'
+            config: |
+              [prometheus_remote_write:https:my_instance]
+                  enabled = yes
+                  destination = example.domain:example_port
```

Comment:
Above you used IP and port as example, I suggest to do the same here in the example.